src/util.cpp

```diff
@@ -385,8 +385,15 @@ std::string GetArg(const std::string& strArg, const std::string& strDefault)
 
 int64_t GetArg(const std::string& strArg, int64_t nDefault)
 {
-    if (mapArgs.count(strArg))
-        return atoi64(mapArgs[strArg]);
+    if (mapArgs.count(strArg)) {
+        int64_t n;
+        if (ParseInt64(mapArgs[strArg], &n)) {
+            return n;
+        }
+        // Perhaps nDefault should be returned here, but this mimics the
+        // (previously used) atoi64() behavior, and some tests depend on it
+        return 0;
+    }
```

Comment:
Is the "Perhaps..." comment too chatty? To elaborate slight on this comment... My first attempt at this change just returned `nDefault` (if `ParseInt64` return false) but that caused tests to fail! We could change the tests, but we'd diverge from bitcoin. I decided to come down on the side of compatibility, but let me know if you think otherwise.